invenio_banners/services/schemas.py

```diff
@@ -1,22 +1,29 @@
 # -*- coding: utf-8 -*-
 #
-# Copyright (C) 2022 CERN.
+# Copyright (C) 2022-2023 CERN.
 #
 # Invenio-Banners is free software; you can redistribute it and/or modify it
 # under the terms of the MIT License; see LICENSE file for more details.
 
 """Banners schema."""
 
+from datetime import datetime
+
 from invenio_records_resources.services.records.schema import BaseRecordSchema
 from marshmallow import fields
 
 
 class BannerSchema(BaseRecordSchema):
     """Schema for banners."""
 
-    message = fields.String()
-    url_path = fields.String()
-    category = fields.String()
-    start_datetime = fields.DateTime()
-    end_datetime = fields.DateTime()
-    active = fields.Boolean()
+    message = fields.String(required=True)
+    url_path = fields.String(allow_none=True)
+    category = fields.String(required=True)
+    start_datetime = fields.DateTime(
+        required=True,
+        metadata={"default": datetime.utcnow().strftime("%Y-%m-%d %H:%M:%S")},
+    )
+    end_datetime = fields.DateTime(allow_none=True)
+    active = fields.Boolean(required=True, metadata={"default": True})
+    created = fields.DateTime(dump_only=True)
```

Comment:
Have a look to what @Pineirin has done [here](https://github.com/inveniosoftware/invenio-pages/blob/master/invenio_pages/services/schemas.py#L25):
`created = TZDateTime(timezone=timezone.utc, format="iso", dump_only=True)`